pkgs/profile-python3/default.nix

```
@@ -0,0 +1,29 @@
+# ============================================================================ #
+#
+#
+#
+# ---------------------------------------------------------------------------- #
+
+{ lib, bash, coreutils, system }: let
+
+# ---------------------------------------------------------------------------- #
+
+  mkProfileLocal = args:
+    ( import ../../lib/mk-profile-util.nix ).mkProfileLocal ( {
```

This should be in `lib`

---

I need to pass in packages, so this wraps the lib routine

---

I'm not quite sure what you mean, I mean I would expect this to eventually look like
```suggestion
    lib.mkProfileLocal ( {
```
I don't think you've setup your lib files to play nicely with capacitor though because when I `flox eval .#lib` I get
```
'Function called without required argument "bash" at /nix/store/3rcbj8ygx2cj1rh41kxdqj9jhdsshmzk-source/lib/mk-profile-util.nix:24
```

---

What is the correct way to add these to to `floxpkgs`?

I spent 10 minutes reading the Capacitor docs and trying to figure it out but I'm not making real progress.